### DISCUSSION TITLE
How to displays comments form

### DISCUSSION THREAD

**Discussion by mveer08** [colbyfayock/next-wordpress-starter#401] [General]
How to display new comment form and also to show existing comments?

**Comment by colbyfayock** [comment]
there's  currently not a way to do this in the starter

that said if you'd like to build it in for your project, you can query the comments via WPGraphQL: https://www.wpgraphql.com/docs/comments/

the above should also reference how you can create comments, so you can create an HTML form and use WPGraphQL to allow people to post as well
